src/psyclone/tests/psyir/nodes/intrinsic_call_test.py

```diff
@@ -663,14 +663,14 @@ def test_intrinsic_canonicalisation_not_implemented_errors():
     # names.
     intrinsic = IntrinsicCall(IntrinsicCall.Intrinsic.ALLOCATED)
     intrinsic.addchild(Reference(DataSymbol("a", INTEGER_TYPE)))
-    intrinsic.argument_names
+    _ = intrinsic.argument_names
     intrinsic._argument_names[0] = (intrinsic._argument_names[0][0], "array")
     with pytest.raises(NotImplementedError) as err:
         intrinsic.canonicalise()
     assert ("Cannot canonicalise 'ALLOCATED' as non-optional argument name "
             "'array' found but the Intrinsic has context-sensitive argument "
             "names which is unsupported by PSyclone. Supplied intrinsic was "
```

Comment:
Seeing this in context I realise that we already have the name of the intrinsic at the start of the message so you can remove the final sentence.

Reply:
Ah yeah, removed.